ui/components/ui/box/box.js

```diff
@@ -10,6 +10,7 @@ import {
   JUSTIFY_CONTENT,
   SIZES,
   TEXT_ALIGN,
+  FLEX_DIRECTION,
```

Comment:
@brad-decker I added flexDirection as a prop on `Box` because I was wanting to use Box as containers for elements that should stack in columns. Objections?